Riot/Modules/Room/Views/BubbleCells/Call/CallBubbleCellBaseContentView.swift

```diff
@@ -0,0 +1,122 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import Reusable
+
+class CallBubbleCellBaseContentView: UIView {
+    
+    @IBOutlet private weak var paginationTitleView: UIView!
+    @IBOutlet private weak var paginationLabel: UILabel!
+    @IBOutlet private weak var paginationSeparatorView: UIView!
```

Comment:
Pagination views are already in `BubbleCellContentView.xib`. You can look at `KeyVerificationBaseBubbleCell.render`

Reply:
As i'm not using `BaseBubbleCell`, using `BubbleCellContentView` is not suitable for call tiles. These are not real bubbles.